applications/Misc/flipbip/manifest.yml

```diff
@@ -0,0 +1,17 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/xtruan/FlipBIP.git
+    commit_sha: e19ecc0145e2e63035682209f84e5ffd4fa4b2a7
+description: "Crypto wallet"
+changelog: "v1.10.0"
```

Comment:
this is version number, not a changelog - a changelog should include at least 1 entry like "1.10 - initial release"